openfe/tests/protocols/openmm_md/test_plain_md_protocol.py

```diff
@@ -180,6 +181,27 @@ def test_dry_run_gaff_vacuum(benzene_vacuum_system, tmpdir):
         system = unit.run(dry=True)["debug"]["system"]
 
 
+@pytest.mark.skipif(not HAS_ESPALOMA, reason='espaloma is not available')
+def test_dry_run_espaloma_vacuum(benzene_vacuum_system, tmpdir):
```

Comment:
Ideally, we should check more things, like if the partial charges made it through, etc.. I'm ok with this as a smoke test, but we should raise a separate issue to try it out in a production setting.